src/raftstore/store/store.rs

```diff
@@ -151,6 +151,8 @@ pub struct Store<T, C: 'static> {
     pub apply_worker: Worker<ApplyTask>,
     apply_res_receiver: Option<StdReceiver<ApplyTaskRes>>,
 
+    last_compact_checked_key: Option<Key>,
```

Comment:
Why not just use `Key` and initialize it with `DATA_MIN_KEY`?

Reply:
We still need handle None after we have finished a round.